fachinfo-bw/.htaccess

```
@@ -1,9 +1,9 @@
 # .htaccess for W3ID redirection
-# This file redirects from https://w3id.org/fachinfo-bw to https://glockenturm.github.io/skohub-pages/
+# This file redirects from https://w3id.org/fachinfoustgbw to glockenturm.github.io/fachinfoustgbw/
 
 # Contact information
 # Maintainer: Glockenturm
 # GitHub: https://github.com/Glockenturm
 
 RewriteEngine on
-RewriteRule ^(.*) https://glockenturm.github.io/skohub-pages/$1 [R=302,L]
+RewriteRule ^(.*) glockenturm.github.io/fachinfoustgbw/$1 [R=302,L]
```

Was dropping the `https://` prefix intentional?
```suggestion
RewriteRule ^(.*) https://glockenturm.github.io/fachinfoustgbw/$1 [R=302,L]
```